site/EmbedChart.tsx

```diff
@@ -9,16 +9,20 @@ import {
 } from "mobx"
 import { Grapher } from "../grapher/core/Grapher"
 import { GrapherFigureView } from "./GrapherFigureView"
-import { splitURLintoPathAndQueryString } from "../clientUtils/url"
 import { deserializeJSONFromHTML } from "../clientUtils/serializers"
+import { Url } from "../clientUtils/urls/Url"
+import { excludeUndefined } from "../clientUtils/Util"
 
 @observer
 export class EmbedChart extends React.Component<{ src: string }> {
+    @computed private get url(): Url {
+        return Url.fromURL(this.props.src)
+    }
     @computed private get configUrl() {
-        return splitURLintoPathAndQueryString(this.props.src).path
+        return excludeUndefined([this.url.base, this.url.pathname]).join("")
```

Comment:
Yeah, I think there should be a getter – I was looking for a good name for something that discards `queryStr` and `hash` and returns just the ______ URL 😄

Reply:
I just created a `Url.baseAndPath` getter, do you think that makes sense?

Reply:
Thinking about it now – `base` is a bad name, because `BAKED_BASE_URL` contains both `.base` and `.path`. Should we rename `base` to `root` maybe?

Reply:
I think it would make sense to take inspiration from the [URL browser API](https://developer.mozilla.org/en-US/docs/Web/API/URL#properties).
They basically have
```
origin pathname search hash
   ^
   |
   protocol + hostname + port
```

So I think `origin` would be a good name to go with rather than `base`.

Reply:
Awesome! Renamed `base` to `origin`.